Add post about link_package, configure_repositories and tokens in UI

saraycp opened the pull request.

dmarcoux commented on _includes/partials/_series-of-posts-about-scm-integration.md:
```diff
@@ -1,2 +1,2 @@
 _We started off the [continuous integration between OBS and GitHub/GitLab](/2021/05/31/scm-integration/) in May 2021,
-then [made some improvements](/2021/06/14/scm-integration-improvements/) in June 2021._
+then [made some improvements](/2021/06/14/scm-integration-improvements/) in June 2021. We introduced advanced features like [reporting filters and support for self-hosted SCM](https://openbuildservice.org/2021/07/12/scm-integration-more-improvements) together with a list of [common pitfalls](https://openbuildservice.org/2021/07/12/scm-integration-pitfalls) in July 2021_
```
We need to use relative links instead of absolute links.

I think we should also link here to the blog post we're adding in this PR. Maybe something like:

```suggestion
then [made some improvements](/2021/06/14/scm-integration-improvements/) in June 2021. We introduced advanced features like [reporting filters and support for self-hosted SCM](/2021/07/12/scm-integration-more-improvements) together with a list of [common pitfalls](/2021/07/12/scm-integration-pitfalls) in July 2021_ and now, we added [two steps and a UI for tokens](/2021/08/02/link-package-configure-repositories-and-ui).
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,115 @@
+Starting today, you can set up your SCM to make OBS create a package linked to an existing one.
+You can also configure the repositories you want to build your package for.
+And as if that weren’t enough, you are now able to manage the tokens through the web UI.
+Join [the beta program](https://openbuildservice.org/2018/10/04/the-beta-program/) and keep reading to discover all you can do.
```
We should use relative links instead of absolute links:

```suggestion
Join [the beta program](/2018/10/04/the-beta-program/) and keep reading to discover all you can do.
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,115 @@
+
+## Tokens on the Web User Interface
+
+In our [first blog post] https://openbuildservice.org/2021/05/31/scm-integration/ regarding the SCM integration with OBS, we explained how to handle workflow tokens via API.
```
Same here, relative links instead of absolute links.
```suggestion
In our [first blog post](/2021/05/31/scm-integration) regarding the SCM integration with OBS, we explained how to handle workflow tokens via API.
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+Simply indicate which repositories you want to build your packages for in the SCM configuration file. Easy peasy!
+
+- Set up the integration between GitHub/GitLab and OBS if you have not yet. https://openbuildservice.org/2021/05/31/scm-integration/
```
```suggestion
- Set up the integration between GitHub/GitLab and OBS. If you have not yet, please refer to this [blog post](/2021/05/31/scm-integration/).
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+OBS moves forward on its integration with GitHub/GitLab by adding two new steps: `link_package` and `configure_repositories`.
+Starting today, you can set up your SCM to make OBS create a package linked to an existing one.
+You can also configure the repositories you want to build your package for.
+And as if that weren’t enough, you are now able to manage the tokens through the web UI.
```
```suggestion
And as if this wasn’t enough, you are now able to manage the tokens through the web UI.
```

saraycp replied:
I took the sentence from the [Collins Dictionary](https://www.collinsdictionary.com/dictionary/spanish-english/y-por-si-fuera-poco), I don't know if the version with `wasn't` is also valid.

dmarcoux replied:
They are both valid, but the version with `weren't` is more formal. It depends on the the tone you want, I would say it's not formal when reading other sentences in the blog post.

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+Branching packages is not enough for you? Let's go step by step 😉  
+OBS moves forward on its integration with GitHub/GitLab by adding two new steps: `link_package` and `configure_repositories`.
+Starting today, you can set up your SCM to make OBS create a package linked to an existing one.
```
```suggestion
Starting today, you can set up your OBS workflows to create a package linked to an existing one.
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+Branching packages is not enough for you? Let's go step by step 😉  
+OBS moves forward on its integration with GitHub/GitLab by adding two new steps: `link_package` and `configure_repositories`.
+Starting today, you can set up your SCM to make OBS create a package linked to an existing one.
+You can also configure the repositories you want to build your package for.
```
```suggestion
You can also configure the repositories of a project in which you want to build packages.
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+As we strive to make your work easier, from now on you can manage any kind of OBS tokens through the web interface.
+
+Go to your profile page and click on the action `Manage Your Tokens` on the left-hand side menu.
```
```suggestion
Go to your profile page and click on the action `Manage Your Tokens` in the left-hand side menu.
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+  <figcaption>Managing tokens in the web UI</figcaption>
+</figure>
+
+For now, only workflow tokens can be updated to change the SCM token attached to them. This is in case you had to regenerate the SCM token if you for example lost it.
```
I wrote this in the Etherpad, but just realized that I forgot commas... oopsie!

```suggestion
For now, only workflow tokens can be updated to change the SCM token attached to them. This is in case you had to regenerate the SCM token if you, for example, lost it.
```

saraycp replied:
no problem

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+Follow these easy steps for that:
+
+- Set up the integration between GitHub/GitLab and OBS if you haven't yet. https://openbuildservice.org/2021/05/31/scm-integration/
```
```suggestion
- Set up the integration between GitHub/GitLab and OBS. If you have not yet, please refer to this [blog post](/2021/05/31/scm-integration/).
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
````diff
@@ -0,0 +1,116 @@
+        source_package: ctris
+```
+
+Providing the source project `games` and the source package `ctris` OBS will link the package `home:$OBS_USER:games:PR-$PR_NUMBER/ctris` to `games/ctris`.
````
```suggestion
Providing the source project `games` and the source package `ctris`, OBS will link the package `home:$OBS_USER:games:PR-$PR_NUMBER/ctris` to `games/ctris`.
```

saraycp replied:
done

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+## The `configure_repositories` Step
+
+You can not imagine how versatile the workflows could be with this new step!
```
Since this is not related to a specific set of workflows, but just workflows in general, we should remove `the`:

```suggestion
You can not imagine how versatile workflows could be with this new step!
```

saraycp replied:
ok!

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+You can not imagine how versatile the workflows could be with this new step!
+
+Simply indicate which repositories you want to build your packages for in the SCM configuration file. Easy peasy!
```
```suggestion
Simply indicate which repositories you want for a project in which you'll build your packages for in the SCM configuration file. Easy peasy!
```

saraycp replied:
I rephrased it, let me know what you think, please:

`Inside the SCM configuration file, simply indicate which repositories you want for a project in which you'll build your packages for. Easy peasy!`

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+**NOTE:**  
+**The source project** has to be a project which was a source project from a previous step like `link_package` or `branch_package`.  
+**The target project** is going to be named after the source project, `home:$OBS_USER:$SOURCE_PROJECT_NAME:PR-$PR_NUMBER`, i.e. `home:Iggy:games:PR-14`.
```
I'm not sure if this is clear enough that we're not referring to `target_project` under `repositories`.

saraycp replied:
I rephrase it, please have a look

eduardoj commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+Follow these easy steps for that:
+
+- Set up the integration between GitHub/GitLab and OBS if you haven't yet. https://openbuildservice.org/2021/05/31/scm-integration/
```
Maybe a link is missed here?

saraycp replied:
true! I fixed two of them

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
```diff
@@ -0,0 +1,116 @@
+
+## More to Come!
+
+Do not miss the upcoming blog posts, there are a lot of features to come:
```
Same as my previous comment above specific vs generic:

```suggestion
Do not miss upcoming blog posts, there are a lot of features to come:
```

saraycp replied:
true!

dmarcoux commented on _posts/development/2021-08-02-link-package-configure-repositories-and-ui.md:
````diff
@@ -0,0 +1,116 @@
+```
+workflow:
+  steps:
+    - configure_repositories:
````
This step won't work by itself, as you wrote below under **NOTE:**.

So should we have `branch_package` in this example? This way the example would actually work.

saraycp replied:
ok! I'll update the YAML.

dmarcoux reviewed (changes requested):
Really nice writing :+1:, I have a few suggestions to make this even better!

saraycp commented:
@eduardoj and @dmarcoux thanks for your feedback. I already addressed all the suggestions except [this one](https://github.com/openSUSE/obs-landing/pull/307#discussion_r680925064).

I also decided to rephrase a few sentences:
https://github.com/openSUSE/obs-landing/pull/307#discussion_r680925695
https://github.com/openSUSE/obs-landing/pull/307#discussion_r680925838

dmarcoux commented on _includes/partials/_series-of-posts-about-scm-integration.md:
```diff
@@ -1,2 +1,2 @@
 _We started off the [continuous integration between OBS and GitHub/GitLab](/2021/05/31/scm-integration/) in May 2021,
-then [made some improvements](/2021/06/14/scm-integration-improvements/) in June 2021._
+then [made some improvements](/2021/06/14/scm-integration-improvements/) in June 2021. We introduced advanced features like [reporting filters and support for self-hosted SCM](/2021/07/12/scm-integration-more-improvements) together with a list of [common pitfalls](/2021/07/12/scm-integration-pitfalls) in July 2021_ and now, we added [two steps and a UI for tokens](/2021/08/02/link-package-configu ...
```
Removing one unneeded underscore:

```suggestion
then [made some improvements](/2021/06/14/scm-integration-improvements/) in June 2021. We introduced advanced features like [reporting filters and support for self-hosted SCM](/2021/07/12/scm-integration-more-improvements) together with a list of [common pitfalls](/2021/07/12/scm-integration-pitfalls) in July 2021 and now, we added [two steps and a UI for tokens](/2021/08/02/link-package-configure-repositories-and-ui)._
```

saraycp replied:
done

saraycp commented:
I've just updated the GIF to include the edit/update token part too.
